docs/ja/faq.md

```
@@ -1,49 +1,47 @@
-GeonicDB に関するよくある質問と回答をまとめたものです。
+GeonicDB に関するよくある質問と回答のコレクションです。
 
 ## 目次
 
-- [データ量とパフォーマンス](#データ量とパフォーマンス)
-- [FIWARE Orion との違い](#fiware-orion-との違い)
-- [デプロイと運用](#デプロイと運用)
-- [API の使い方](#api-の使い方)
-- [地理空間拡張機能](#地理空間拡張機能)
-- [セキュリティ](#セキュリティ)
+- [データ量とパフォーマンス](#data-volume-and-performance)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)